Makefile

```
@@ -44,6 +52,10 @@ lint: ## Lint webserver source files
 tidy: ## Tidy webserver source files
 	clang-tidy srcs/**/*.hpp srcs/**/*.cpp -fix
 
+.PHONY: syntax
+syntax: ## Check syntax of source files
+	${CXX} -fsyntax-only srcs/**/*.hpp $(SRCS)
```

```suggestion
	$(CXX) -fsyntax-only srcs/**/*.hpp $(SRCS)
```

`{}`で囲うと意味が違うんでしたっけ？

---

`()`で囲っているつもりでした、、、